docs/runtimes/databricks.ipynb

```diff
@@ -4,7 +4,6 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        # an illegal artifact does not raise an error, it logs an error log instead, for example:
</pre><p>Fix indentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4979/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>